### DISCUSSION TITLE
One to None/One/Many Relationship

### DISCUSSION THREAD

**Discussion by Nasifshuvo** [prisma/prisma#15274] [Q&A] [answered]
In this relation a user can have many posts and every post must have one user.

```
model User {
  id        Int      @id @default(autoincrement())
  posts     Post[]
}
model Post {
  id         Int     @id @default(autoincrement())
  author     User    @relation(fields: [authorId], references: [id])
  authorId   Int
}

```
There can be a user without any post. But I can not create any user without connecting any post. Please help me to find a better way to solve my problem. I want to create user with/without post.

**Comment by austincrim** [comment, marked as answer]
Hey @Nasifshuvo 👋,

You should be able to create a user with an empty `posts` array.

```ts
await prisma.user.create({
  data: {
    posts: { create: [] }
  }
})
```
